### ISSUE TITLE
How do we stop listening?

### ISSUE THREAD

**Issue by Prutya** [MatthieuLemoine/push-receiver#7]

**Comment by MatthieuLemoine** [comment]
As of now, there's no way to stop listening to incoming notifications.

**Comment by ibash** [comment]
@MatthieuLemoine this leads to a crash in the case where the WebContents is destroyed and then electron-push-receiver tries to send it a push notification. Going to send a PR.

**Comment by MatthieuLemoine** [comment]
Once #13 merged, you could call `client.destroy` to stop listening : 
```
const client = await listen(credentails, callback);
client.destroy()
```

**Comment by Prutya** [comment]
Awesome! Thank you

**Comment by MatthieuLemoine** [comment]
Released in v2.0.0
